cmd/kubeadm/app/util/env.go

```diff
@@ -41,3 +41,17 @@ func GetProxyEnvVars() []v1.EnvVar {
 	}
 	return envs
 }
+
+// MergedEnv merge the extraEnvs with proxyEnv, extraEnvs take precedence over the proxyEnv.
```

Comment:
func should be called with a verb. MergeEnv

Reply:
it should be a generic, variadic argument function that takes a slice if env slices and makes the last slices override / take precedence.

Reply:
Done, pls let me know if the current version address your comment.